tests/kola/misc/test-status.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash     
+
+# Simple sanity checks for systemd unit status updates.
+
+set -xeuo pipefail
+
+. ${KOLA_EXT_DATA}/libtest.sh
+
+cd $(mktemp -d)
+
+systemctl stop zincati.service
+echo "updates.enabled = false" > /etc/zincati/config.d/disable-in-test.toml
```

Comment:
Please add a numeric prefix to this, possibly `99-test-status-updates-enabled.toml`.